File: 231219/2개 이상의 알파벳/more-than-one-alphabet.cpp
#include <iostream>
#include <string>
using namespace std;

bool isMoreThanTwo(string str, int n) {
    char alp = str[0];
    int cnt = 0;
    for (int i = 1; i < n; i++) {
        if (str[i] != alp) return true;
    }
    return false;
}

int main() {
    string strA = "";
    cin >> strA;

    if(isMoreThanTwo(strA, strA.length())) cout << "Yes" << endl;
    else cout << "No" << endl;
    // 여기에 코드를 작성해주세요.
    return 0;
}

File: 250312/2개 이상의 알파벳/more-than-one-alphabet.py
def is_more_than_two(str):
    if len(str) > 1:
        for i in range(1, len(str) - 1):
            if str[i] != str[i + 1]:
                return True
        
    return False

str = input()

if is_more_than_two(str):
    print("Yes")
else:
    print("No")